package.json

```diff
@@ -70,6 +70,11 @@
 					"type": "boolean",
 					"default": true,
 					"description": "Forces indenting with two spaces when Dart files are opened. This is on by default because VS Code doesn't support per-language settings and most people use tabs/4 spaces for other languages."
+				},
+				"dart.showTodos": {
+					"type": "boolean",
+					"default": false,
```

Comment:
What about collapsing the TODOs into a single line?

> You have 123 TODOs. Set dart.showTodos in settings to have them displayed here

I think having one item in the list won't be a problem and it aids discovery. If we can hook the user clicking on it (I suspect not) we could even do "click to show" and then fill them in.